src/__swaps__/screens/Swap/resources/search/search.ts

```diff
@@ -102,6 +157,43 @@ async function tokenSearchQueryFunction({
   }
 }
 
+async function tokenSearchQueryFunctionAllNetworks({ queryKey: [{ query }] }: QueryFunctionArgs<typeof tokenSearchAllNetworksQueryKey>) {
+  const queryParams: {
```

Comment:
the query params are simpler now as the new v3 endpoint consolidates for us - we don't need to make a separate call per each "type" of list (e.g, verified, high liquidity, unverified"